README.md

```diff
@@ -1,5 +1,7 @@
 # Open Source Party Declaration of Principle
 
+![alt tag](https://github.com/opensourceparty/Open-Source-Party/img/flag.jpg)
```

Comment:
Can you use a relative link to the image here instead of full path?

It should be just `![alt tag](img/flag.jpg)`

https://help.github.com/articles/relative-links-in-readmes/